packages/kbn-test/src/functional_tests/tasks.ts

```diff
@@ -108,7 +108,13 @@ export async function runTests(options: RunTestsParams) {
 
         let es;
         try {
+          if (process.env.ES_BOOT_UP_SLOWDOWN) {
+            await new Promise((r) => setTimeout(r, 60 * 1000));
```

Comment:
`import { setTimeout } from 'timers/promises'` 😉

Reply:
I removed slowdowns since we didn't see much effect on duration distribution. I can add it and use `timers/promises` if you think we need :)